### DISCUSSION TITLE
Design for interne applikasjoner

### DISCUSSION THREAD

**Discussion by sergio-haisch** [digdir/designsystemet#1687] [Feature requests]
Det designes en del løsninger for super brukere og interne applikajsoner. 
Hva er best practises innfor dette?

**Comment by fredrikstai** [comment]
Godt forslag.

Det er åpenbart noen forskjeller mellom selvbetjening for innbyggere og applikasjoner som interne brukere bruker i opp til flere timer om dagen. Det kan være behov for større informasjonstetthet, annen type navigasjon og informasjonsarkitektur, fleksibel layout, snarveier…

Det er forresten flere gode "god praksis"-artikler om interne flater på Aksel: https://aksel.nav.no/god-praksis/interne-flater

**Comment by Camulos** [comment]
Relates to #3490
